src/styles/header.scss

```diff
@@ -0,0 +1,119 @@
+.header {
+  @include desktop {
+    max-width: 1280px;
+  }
+
+  @include big-desktop {
+    max-width: 1440px;
+
+  }
+
+  &__nav {
+    padding: 15px 0 20px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    @include tablet {
+      padding: 20px 0 30px;
+    }
+
+    @include desktop {
+      max-width: 1280px;
+      margin: auto;
+      padding: 50px 0 90px;
+    }
+  }
+
+  &__menu {
+    @include desktop {
+      display: none;
+    }
+  }
+
+  &__logotipe {
+    display: none;
+
+    @include desktop {
+      display: flex;
+      align-items: center;
+    }
+  }
+
+  &__logo {
+    @include desktop {
+      display: none;
+    }
+  }
+
+  &__bar {
+    display: flex;
+    align-items: center;
+    gap: 292px;
+    font-size: 18px;
+    font-weight: 500;
+  }
+
+  &__list {
+    display: none;
+
+    @include desktop {
+      display: flex;
+      gap: 40px;
+      color: $text-minor-color;
+      font-weight: 500;
+      line-height: 150%;
+      text-transform: capitalize;
+    }
+  }
+
+  &__item {
+    @include desktop {
+      position: relative;
+
+      &:hover .header__link::after {
+        opacity: 1;
+        transform: scaleX(1);
+      }
+    }
+  }
+
+  &__link {
+    @include desktop {
+      position: relative;
+
+      &::after {
+        content: "";
+        position: absolute;
+        right: 100%;
+        left: 0;
+        bottom: -4px;
+        width: 100%;
+        height: 1px;
+        background-color: $text-minor-color;
+        opacity: 0;
+        transition: opacity 0.3s ease, transform 0.3s ease;
+        transform: scaleX(0);
+      }
+    }
+  }
+
+  &__bag {
+    background-image: url(../images/icons/shop-mobile.svg);
```

Comment:
Remember to use alt text for all images, even background images, for accessibility reasons. You can use the 'attr' function in CSS to add an alt attribute from the HTML to your background image.